.travis.yml

```diff
@@ -12,19 +13,25 @@ matrix:
       env: TOXENV=pypy
     - env: TOXENV=flake8
     - env: TOXENV=checkspelling
+
 addons:
   apt:
     packages:
     - libtidy-0.99-0
     - aspell
     - aspell-en
+
 install:
   - pip install tox
-  - pip install coveralls
+  - pip install coverage<4.0
```

Comment:
Is there a reason for this (<4.0)?  I'm using coverage 4.0+ on all my projects without issue.

Reply:
See 20b17c29081. And the `.coverage` file format changed in 4.0. As the tox envs output the older format, we need the older version to upload as well.

Reply:
Interesting. I must not be covering 3.2... Or for some reason it doesn't cause an issue for me :/. Now I'm going to need to audit my repos.